libs/shared/ui/nuverial/src/lib/components/file-upload/file-upload.component.ts

```diff
@@ -199,6 +201,9 @@ export class NuverialFileUploadComponent extends FormInputBaseDirective implemen
   @Input()
   public multiple = false;
 
+  @Input()
+  public showLoading = false;
```

Comment:
Lets use the term loading for consistency
```suggestion
  public loading = false;
```